### DISCUSSION TITLE
Gradle sync failed

### DISCUSSION THREAD

**Discussion by tanbirali** [robstoll/atrium#1405] [Contributor Q&A]
Unable to find method ''java.lang.String org.gradle.api.tasks.SourceSet.getCompileConfigurationName()''
'java.lang.String org.gradle.api.tasks.SourceSet.getCompileConfigurationName()'

I am running into this issue when I am trying to open the project in my local system

**Comment by robstoll** [comment]
sounds like you are using gradle 7.x instead of gradlew which uses gradle 6.9.x
are you using intellij? make sure you import the project via the build.gradle.kts

**Reply by robstoll** [reply]
@tanbirali problem solved?

**Comment by tanbirali** [comment]
Yes it's solved I didn't had to do anything IntelliJ fixed it itself 
Actually I opened it on VS Code first that's when I faced that issue
